src/opamWixMain.ml

```diff
@@ -22,7 +22,8 @@ type config = {
   package_guid: string option;
   icon_file : filename option;
   dlg_bmp : filename option;
-  ban_bmp : filename option
+  ban_bmp : filename option;
+  keep_wxs : bool
```

Comment:
```suggestion
  keep_wxs : bool;
```
Like that when another element in the record is added, no change in the line above.